templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -0,0 +1,591 @@
+        {%- endif -%}
+        {%- if slot == 'description' -%}
+        <div class="p-section--shallow">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/c3405735-who-does-the-cra-apply-to.png",
+          alt="",
+          width="1832",
+          height="765",
+          hi_def=True,
+          loading="auto|lazy"
+          ) | safe
+          }}
+        </div>
+        {%- endif -%}
+  
+        {%- if slot == 'list_item_title_1' -%}
+          <h3 class="p-heading--5">Manufacturers</h3>
+        {%- endif -%}
+        {%- if slot == 'list_item_description_1' -%}
+          <p>Entities that produce and deliver PDEs to consumers in the EU market.</p>
+        {%- endif -%}
+  
+        {%- if slot == 'list_item_title_2' -%}
+          <h3 class="p-heading--5">Providers</h3>
+        {%- endif -%}
+        {%- if slot == 'list_item_description_2' -%}
+          <p>Entities that provide components or software (whether open source or proprietary) used by manufacturers.</p>
+        {%- endif -%}
+  
+        {%- if slot == 'list_item_title_3' -%}
+          <h3 class="p-heading--5">Importers</h3>
+        {%- endif -%}
+        {%- if slot == 'list_item_description_3' -%}
+          <p>Entities that import or distribute PDEs marketed in the EU.</p>
+        {%- endif -%}
+  
+      {%- endcall -%}
+    </div>
+
+    <hr class="p-rule is-fixed-width" />
+
+    <div class="u-fixed-width">
+      <section class="p-section">
+        <div class="row--50-50">
+          <div class="col">
+            <h2>Canonical's commitment <br/> to the CRA</h2>
+          </div>
+          <div class="col">
+            <p>We are focusing on making CRA compliance as easy as possible on our entire range of products and services.</p>
+            <p>
+              Canonical has chosen to meet the challenges and requirements of the CRA head-on, allowing all of our customers who consume open source through us to benefit from our commitments to the CRA and focus on building future-proof products.
+            </p>
+            <p>
+              Canonical has committed to:
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked u-hide"></li>
+                <li class="p-list__item is-ticked">Ensuring our operating systems are compliant</li>
+                <li class="p-list__item is-ticked">Completing certification on relevant products</li>
+                <li class="p-list__item is-ticked">Performing attestation for non-critical products</li>
+                <li class="p-list__item is-ticked">Assuming “manufacturer” duties under the CRA.</li>
+              </ul>
+            </p>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <hr class="p-rule is-fixed-width" />
+
+    <div class="u-fixed-width p-strip">
+      <section class="p-strip">
+          <h2>
+            <a aria-controls="contact-modal" class="js-invoke-modal">Need help with your CRA roadmap? Contact our experts ›</a>
```

Comment:
We use character entities for spaces and symbols. Can you please update all of these occurrences?
```suggestion
            <a aria-controls="contact-modal" class="js-invoke-modal">Need help with your CRA roadmap? Contact our experts&nbsp;&rsaquo;</a>
```